dom-querying/main.ts

```diff
@@ -0,0 +1,17 @@
+const hi: string = 'hello, World';
+console.log(hi);
+
+const $element = document.querySelector('h1');
+console.dir($element);
+
+const $element1 = document.querySelector('#explanation');
+console.dir($element1);
+
+const $element2 = document.querySelector('.hint');
+console.dir($element2);
+
+const $element3 = document.querySelectorAll('p');
+console.dir($element3);
+
+const $element4 = document.querySelectorAll('.example-link');
```

Comment:
Your variable names should be indicative of what they are storing. `$element` and `$element1` don't give me any information about what they are actually storing.

Some better examples would be `$h1`, `$paragraphs`, etc..